### PR TITLE
Remove Auth Tokens

### PR DESCRIPTION
**Pull request by ECMAScript3**
You published your GitHub secure and auth tokens, that’s a HUGE security risk. Couldn’t find another way to contact Tufantas about his fork. Please do not merge.

### CODE REVIEW

**Comment by parachvte** [comment]
Seems like you made lots of changes, including your local changes.

Could you make a clean PR for you changes?

File: .travis.yml [hunk 1]
```diff
@@ -107,4 +107,4 @@ deploy:
         # https://docs.travis-ci.com/user/deployment#Conditional-Releases-with-on
         tags: true  # Deploy app only when a tag is applied to the commit
         node: '10'
-        repo: Snapmaker/Luban
+        repo: tufantas/T3D-Suite
```
Comment (parachvte) [inline .travis.yml:110]: Your local changes.

Comment (ECMAScript3) [reply .travis.yml:110]: I’m not trying to merge this branch, the author of the branch published his GitHub auth token in the source code and I have no other way to notify him.

Comment (parachvte) [reply .travis.yml:110]: His repo has not been updated since 7 month ago. Maybe a fork to his repo will notify him, or @tufantas.

**Pull request closed without merging**